.github/workflows/expotools.yml

```diff
@@ -22,21 +22,29 @@ jobs:
     steps:
       - name: 👀 Checkout
         uses: actions/checkout@v3
+
+      - name: ⬢ Setup Bun
+        uses: oven-sh/setup-bun@v1
+        with:
+          bun-version: 1.x
+
       - name: ♻️ Restore caches
         uses: ./.github/actions/expo-caches
         id: expo-caches
-        with:
-          yarn-tools: 'true'
```

Comment:
Note how I disabled the cache here, Bun installs faster than Github Actions can untar the cache... 😂